src/clue.js

```diff
@@ -1,126 +1,186 @@
+  occupation: "Retired Football player"
+};
 
 // Weapons
 
 var rope = {
-  name: 'rope',
+  name: "rope",
   weight: 10
 };
 var knife = {
-  name: 'knife',
+  name: "knife",
   weight: 8
 };
 var candlestick = {
-  name: 'candlestick',
+  name: "candlestick",
   weight: 2
 };
 var dumbbell = {
-  name: 'dumbbell',
+  name: "dumbbell",
   weight: 30
 };
 var poison = {
-  name: 'poison',
+  name: "poison",
   weight: 2
 };
 var axe = {
-  name: 'axe',
+  name: "axe",
   weight: 15
 };
 var bat = {
-  name: 'bat',
+  name: "bat",
   weight: 13
 };
 var trophy = {
-  name: 'trophy',
+  name: "trophy",
   weight: 25
 };
 var pistol = {
-  name: 'pistol',
+  name: "pistol",
   weight: 20
 };
 
 // Rooms
-var DinningRoom = {name: 'Dinning Room'}
-var Conservatory = {name: 'Conservatory'}
-var Kitchen = {name: 'Kitchen'}
-var Study = {name: 'Study'}
-var Library = {name: 'Library'}
-var BilliardRoom = {name: 'Billiard Room'}
-var Lounge = {name: 'Lounge'}
-var Ballroom = {name: 'Ballroom'}
-var Hall = {name: 'Hall'}
-var Spa = {name: 'Spa'}
-var LivingRoom = {name: 'Living Room'}
-var Observatory = {name: 'Observatory'}
-var Theater = {name: 'Theater'}
-var GuestHouse = {name: 'Guest House'}
-var Patio = {name: 'Patio'}
-
+var DinningRoom = { name: "Dinning Room" };
+var Conservatory = { name: "Conservatory" };
+var Kitchen = { name: "Kitchen" };
+var Study = { name: "Study" };
+var Library = { name: "Library" };
+var BilliardRoom = { name: "Billiard Room" };
+var Lounge = { name: "Lounge" };
+var Ballroom = { name: "Ballroom" };
+var Hall = { name: "Hall" };
+var Spa = { name: "Spa" };
+var LivingRoom = { name: "Living Room" };
+var Observatory = { name: "Observatory" };
+var Theater = { name: "Theater" };
+var GuestHouse = { name: "Guest House" };
+var Patio = { name: "Patio" };
 
 // Characters Collection
-var charactersArray = [drOrchid, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
 
 // Rooms' Collection
-var roomsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
+var roomsArray = [
+  DinningRoom,
+  Conservatory,
+  Kitchen,
+  Study,
+  Library,
+  BilliardRoom,
+  Lounge,
+  Ballroom,
+  Hall,
+  Spa,
+  LivingRoom,
+  Observatory,
+  Theater,
+  GuestHouse,
+  Patio
+];
 
 // Weapons Collection
-var weaponsArray = [DinningRoom, Conservatory, Kitchen, Study, Library, BilliardRoom, Lounge, Ballroom, Hall, Spa,  LivingRoom, Observatory, Theater, GuestHouse, Patio];
+var weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+function randomSelector(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
+
+function pickMistery() {
+  // here I delete "var" before misteryEnvelope to make it global for further purpose. althoug I think there is better/cleaner way
```

Comment:
In this case, its fine because the name of the variable is very specific, but you're right. its not a good practice to declare global variables